src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectServices/VsManagedLanguagesProjectSystemServices.cs

```diff
@@ -230,6 +230,12 @@ private static VersionRange ToVersionRange(string version, bool isCpvmEnabled)
                 // Projects that have their packages managed centrally will not have Version metadata on PackageReference items.
                 return null;
             }
+            else if (string.IsNullOrEmpty(version))
```

Comment:
Creating a variable for this would prevent 2 checks on nearly every invocation of `ToVersionRange`.

Reply:
or refactor to only check once:
```cs
if (string.IsNullOrEmpty(version)
{
    if (isCpvmEnabled)
    {
        // copy the comment
        return null;
    }
    else
    {
        return VersionRange.All;
    }
}